src/components/chip/index.tsx

```diff
@@ -202,7 +203,7 @@ const Chip = ({
       >
         <Icon
           source={dismissIcon}
-          tintColor={dismissColor}
+          tintColor={dismissColor || Colors.$iconDefault}
```

Comment:
Why not set`Colors.$iconDefault` as the 'dismissColor' prop's default value?

Reply:
Fixed :)